### ISSUE TITLE
The latest 0.6.9 release still required the submodule biod/undeaD but it doesn't exist

### ISSUE THREAD

**Issue by twhitehead** [biod/sambamba#397]
As the title says.

```bash
$ git clone -b v0.6.9 --recurse-submodules git@github.com:biod/sambamba.git
```
```
Cloning into 'sambamba'...
remote: Enumerating objects: 79, done.
remote: Counting objects: 100% (79/79), done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 5976 (delta 23), reused 44 (delta 20), pack-reused 5897
Receiving objects: 100% (5976/5976), 3.39 MiB | 5.49 MiB/s, done.
Resolving deltas: 100% (3324/3324), done.
Note: checking out '495567f509d942e01bc77d99d88729c96755a405'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

Submodule 'BioD' (https://github.com/biod/BioD.git) registered for path 'BioD'
Submodule 'htslib' (https://github.com/lomereiter/htslib.git) registered for path 'htslib'
Submodule 'lz4' (https://github.com/lz4/lz4.git) registered for path 'lz4'
Submodule 'undeaD' (https://github.com/biod/undeaD.git) registered for path 'undeaD'
Cloning into '/home/tyson/sambamba/BioD'...
remote: Enumerating objects: 2731, done.        
remote: Total 2731 (delta 0), reused 0 (delta 0), pack-reused 2731        
Receiving objects: 100% (2731/2731), 2.66 MiB | 7.74 MiB/s, done.
Resolving deltas: 100% (1625/1625), done.
Cloning into '/home/tyson/sambamba/htslib'...
remote: Enumerating objects: 5887, done.        
remote: Total 5887 (delta 0), reused 0 (delta 0), pack-reused 5887        
Receiving objects: 100% (5887/5887), 4.37 MiB | 8.46 MiB/s, done.
Resolving deltas: 100% (3930/3930), done.
Cloning into '/home/tyson/sambamba/lz4'...
remote: Enumerating objects: 24, done.        
remote: Counting objects: 100% (24/24), done.        
remote: Compressing objects: 100% (20/20), done.        
remote: Total 10613 (delta 8), reused 13 (delta 4), pack-reused 10589        
Receiving objects: 100% (10613/10613), 4.67 MiB | 9.87 MiB/s, done.
Resolving deltas: 100% (7435/7435), done.
Cloning into '/home/tyson/sambamba/undeaD'...
remote: Repository not found.
fatal: repository 'https://github.com/biod/undeaD.git/' not found
fatal: clone of 'https://github.com/biod/undeaD.git' into submodule path '/home/tyson/sambamba/undeaD' failed
Failed to clone 'undeaD'. Retry scheduled
Cloning into '/home/tyson/sambamba/undeaD'...
remote: Repository not found.
fatal: repository 'https://github.com/biod/undeaD.git/' not found
fatal: clone of 'https://github.com/biod/undeaD.git' into submodule path '/home/tyson/sambamba/undeaD' failed
Failed to clone 'undeaD' a second time, aborting
```

**Comment by pjotrp** [comment]
undead is now part of BioD so it should not be a submodule.

**Comment by pjotrp** [comment]
Was fixed with a9fc5fb86bc4bbd820b33c3ec19629b21e476c76

**Comment by twhitehead** [comment]
Thanks. For anyone else who comes across this, you actually want d67cf50 (one more commit) as a9fc5fb only removed the _.gitmodules_ entry for `undeaD` and not the reference number object leaving the submodules in a broken state.
